src/main/java/wooteco/subway/web/member/interceptor/BearerAuthInterceptor.java

```diff
@@ -1,48 +1,50 @@
 package wooteco.subway.web.member.interceptor;
 
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Component;
 import org.springframework.web.servlet.HandlerInterceptor;
 import org.springframework.web.servlet.ModelAndView;
+
 import wooteco.subway.infra.JwtTokenProvider;
 import wooteco.subway.web.member.AuthorizationExtractor;
 
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
 @Component
 public class BearerAuthInterceptor implements HandlerInterceptor {
-    private AuthorizationExtractor authExtractor;
-    private JwtTokenProvider jwtTokenProvider;
+    private final AuthorizationExtractor authExtractor;
+    private final JwtTokenProvider jwtTokenProvider;
 
     public BearerAuthInterceptor(AuthorizationExtractor authExtractor, JwtTokenProvider jwtTokenProvider) {
         this.authExtractor = authExtractor;
         this.jwtTokenProvider = jwtTokenProvider;
     }
 
     @Override
-    public boolean preHandle(HttpServletRequest request,
-                             HttpServletResponse response, Object handler) {
-        // TODO: Authorization 헤더를 통해 Bearer 값을 추출 (authExtractor.extract() 메서드 활용)
-
-        // TODO: 추출한 토큰값의 유효성 검사 (jwtTokenProvider.validateToken() 메서드 활용)
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
+        String token = authExtractor.extract(request, "bearer");
+        if (!jwtTokenProvider.validateToken(token)) {
```

Comment:
```suggestion
        if (jwtTokenProvider.isInvalidToken(token)) {
```

부정문은 가독성을 해칠 수 있습니다.
Objects.isNull과 Objects.nonNull이 존재하는 이유에 대해 고민해보아요.

Reply:
가독성을 고려하여 부정문을 제거하도록 수정했습니다. :)